bedrock/mozorg/middleware.py

```diff
@@ -45,3 +46,17 @@ def process_view(self, request, view, view_args, view_kwargs):
         else:
             f = super(MozorgRequestTimingMiddleware, self)
             f.process_view(request, view, view_args, view_kwargs)
+
+
+class CrossOriginResourceSharingMiddleware(object):
```

Comment:
Thinking a bit more, maybe not, as it would take more modifications than I'm comfortable with to have their library support per-regex CORS options.